_episodes/02-filedir.md

```diff
@@ -607,18 +607,18 @@ Run `pwd` and `ls -F` to ensure that we're in the directory we expect.
 > The shell interprets the character `~` (tilde) at the start of a path to
 > mean "the current user's home directory". For example, if Nelle's home
 > directory is `/Users/nelle`, then `~/data` is equivalent to
-> `/Users/nelle/data`. This only works if it is the first character in the
+> `/Users/nelle/data`. The tilde only works if it is the first character in the
 > path: `here/there/~/elsewhere` is *not* `here/there/Users/nelle/elsewhere`.
 >
-> Another shortcut is the `-` (dash) character.  `cd` will translate `-` into
+> Another shortcut is the `-` (dash) character. `cd` will translate `-` into
 > *the previous directory I was in*, which is faster than having to remember,
-> then type, the full path.  This is a *very* efficient way of moving back
+> then type, the full path.  This shortcut is a *very* efficient way of moving back
 > and forth between directories. The difference between `cd ..` and `cd -` is
 > that the former brings you *up*, while the latter brings you *back*. You can
 > think of it as the *Last Channel* button on a TV remote.
 {: .callout}
 
-> ## Absolute vs Relative Paths
+> ## Absolute vs. Relative Paths
```

Comment:
We need to get clarification from the style guide on this. For now let's leave it as it was.

Reply:
And the decision is 'vs'.

https://github.com/carpentries/docs.carpentries.org/issues/737#issuecomment-881500785